db/migrate/20151025194751_create_quotes.rb

```diff
@@ -0,0 +1,16 @@
+class CreateQuotes < ActiveRecord::Migration
+  def change
+    create_table :quotes do |t|
+      t.references :stock, index: true, foreign_key: true, null: false
```

Comment:
I think this table should have a composite key of stock,date.

https://github.com/composite-primary-keys/composite_primary_keys